integration/deploy_test.go

```diff
@@ -431,4 +428,14 @@ var _ = Describe("Deploy", func() {
 			defer func(tdf environment.TearDownFunc) { Expect(tdf()).To(Succeed()) }(tearDown)
 		})
 	})
+
+	Context("when the BOSHDeployment cannot be resolved", func() {
+
```

Comment:
```suggestion
```

We're a little inconsistent about newlines, but I think most files don't have a newline between Context and It.